test/server.js

```diff
@@ -109,6 +115,21 @@ function serverTest (t, serverType, serverFamily) {
   })
 }
 
+test('create daemon', function (t) {
+  wrtc = require('electron-webrtc')()
+  wrtc.electronDaemon.once('ready', t.end)
+})
+
+test('websocket server', function (t) {
+  serverTest(t, 'ws', 'inet')
+})
+
+// cleanup
+test('cleanup electron-eval daemon', function (t) {
+  wrtc.close()
+  t.end()
+})
+
```

Comment:
Would prefer to merge these into one `test` block. It's confusing to have state shared across blocks.